Modix/Services/FileUpload/FileUploadHandler.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Serilog;
+
+namespace Modix.Services.FileUpload
+{
+    public class FileUploadHandler
+    {
+        private const ulong ChannelIdToPostModerationLog = 360507591488438283;
```

Comment:
Modix isn't allowed to post to this channel. Before merge we need to either get modix in there or find a different channel